affinity_groups.go

```diff
@@ -156,3 +156,54 @@ type ListAffinityGroupTypesResponse struct {
 	Count             int                 `json:"count"`
 	AffinityGroupType []AffinityGroupType `json:"affinitygrouptype"`
 }
+
+// CreateAntiAffinityGroup represents an anti-affinity Group creation
+type CreateAntiAffinityGroup struct {
+	Name        string `json:"name"`
+	Description string `json:"description,omitempty"`
+	_           bool   `name:"createAntiAffinityGroup" description:"Creates an anti-affinity group"`
+}
+
+// Response returns the struct to unmarshal
```

Comment:
```suggestion
// Response returns an empty structure to unmarshal an Anti-Affinity Group creation API response into.
```

Idem for other `Reponse()` methods in this PR :wink:

Reply:
I wanted to stay consistent with the rest of the file

Reply:
Fair enough 😏